defaults/main.yml

```diff
@@ -2,5 +2,5 @@
 # Pin-Priority of NodeSource repository
 nodejs_nodesource_pin_priority: 500
 
-# 0.10 or 0.12 or 4.x
-nodejs_version: "4.2"
+# 0.10 or 0.12 or 4.x or 5.x
+nodejs_version: "5.0"
```

Comment:
The idea here is you can currently install a specific release, and it is then translated to the 5.x or 4.x required to build the repo path.

We could just get people to enter 0.10, 0.12 or 4.x or 5.x ect.

Actually seems like a better move.